### PR TITLE
Marketplace response bodies notification

### PR DESCRIPTION
**Pull request by Isabella-Veloso**
#### Types of changes
- [ ] New content (endpoints, descriptions or fields from scratch)
- [x] Improvement (make an endpoint's title or description even better)
- [ ] Spelling and grammar accuracy (self-explanatory)

### Changelog
Do not forget to update your changes to our Developer Portal's changelog. Did you create a release note?
- [ ] Yes, I already created a release note about this change.
- [x] No, but I am going to.

### CODE REVIEW

**Review by karenkrieger — CHANGES_REQUESTED** [review]

File: VTEX - Marketplace APIs.json [hunk 1]
```diff
@@ -572,19 +572,22 @@
...
-                                                    "type": "array",
-                                                    "description": "SKUs' ID."
-                                                },
-                                                "Product": {
-                                                    "type": "array",
-                                                    "description": "Product's name."
-                                                }
-                                            }
+                                            "description": "Items and products that belong to sellers.",    
+                                            "items": {
```
Comment (karenkrieger) [inline VTEX - Marketplace APIs.json:578]:
```suggestion
                                            "properties": {
```

File: VTEX - Marketplace APIs.json [hunk 2]
```diff
@@ -572,19 +572,22 @@
...
+                                                "type": "object",          
+                                                "properties": {
+                                                                "Item": {
+                                                                    "type": "array",
+                                                                    "description": "SKUs' ID."
```
Comment (karenkrieger) [inline VTEX - Marketplace APIs.json:583]: Adicionar `items`

File: VTEX - Marketplace APIs.json [hunk 3]
```diff
@@ -572,19 +572,22 @@
...
+                                                                },
+                                                                "Product": {
+                                                                    "type": "array",
+                                                                    "description": "Product's name."
```
Comment (karenkrieger) [inline VTEX - Marketplace APIs.json:587]: Adicionar `items`

**Review by karenkrieger — CHANGES_REQUESTED** [review]

File: VTEX - Marketplace APIs.json (hunk 2 above)
Comment (Isabella-Veloso) [reply VTEX - Marketplace APIs.json]: Na verdade é um "object of objects", então não vou adicionar "items".

File: VTEX - Marketplace APIs.json (hunk 3 above)
Comment (Isabella-Veloso) [reply VTEX - Marketplace APIs.json]: Na verdade é um "object of objects", então não vou adicionar "items".

File: VTEX - Marketplace APIs.json [hunk 4]
```diff
@@ -574,20 +574,21 @@
...
+                                                        "Item": {
+                                                            "type": "array",
+                                                            "description": "SKUs' ID."
                                                                 },
-                                                                "Product": {
-                                                                    "type": "array",
-                                                                    "description": "Product's name."
+                                                        "Product": {
+                                                            "type": "array",
+                                                            "description": "Product's name."
```
Comment (karenkrieger) [inline VTEX - Marketplace APIs.json:589]:
```suggestion
                                                            "description": "Product's name.",
                                                            "items":{
                                                               "type": "string",
                                                               "description": "Product name."
                                                            }
```

File: VTEX - Marketplace APIs.json [hunk 5]
```diff
@@ -574,20 +574,21 @@
...
-                                                                    "description": "SKUs' ID."
+                                            "description": "Items and products that belong to sellers.",
+                                            "required": [
+                                                "Item",
+                                                "Product"
+                                                    ],   
+                                            "properties": {    
+                                                        "Item": {
+                                                            "type": "array",
+                                                            "description": "SKUs' ID."
```
Comment (karenkrieger) [inline VTEX - Marketplace APIs.json:585]:
```suggestion
                                                            "description": "SKUs' ID.",
                                                              "items":{
                                                               "type": "string",
                                                               "description": "SKU ID."
                                                            }
```

**Review by karenkrieger — CHANGES_REQUESTED** [review]

**Review by karenkrieger — APPROVED** [review]
LGTM

**Pull request merged**